gcloud/bigtable/test_column_family.py

```diff
@@ -395,6 +395,60 @@ def test___ne__(self):
         column_family2 = self._makeOne('column_family_id2', None)
         self.assertNotEqual(column_family1, column_family2)
 
+    def _update_test_helper(self, gc_rule=None):
+        from gcloud.bigtable._generated import (
+            bigtable_table_data_pb2 as data_pb2)
+        from gcloud.bigtable._testing import _FakeStub
+
+        project_id = 'project-id'
+        zone = 'zone'
+        cluster_id = 'cluster-id'
+        table_id = 'table-id'
+        column_family_id = 'column-family-id'
+        timeout_seconds = 28
+        table_name = ('projects/' + project_id + '/zones/' + zone +
+                      '/clusters/' + cluster_id + '/tables/' + table_id)
+        column_family_name = table_name + '/columnFamilies/' + column_family_id
+
+        client = _Client(timeout_seconds=timeout_seconds)
+        table = _Table(table_name, client=client)
+        column_family = self._makeOne(column_family_id, table, gc_rule=gc_rule)
+
+        # Create request_pb
+        if gc_rule is None:
+            request_pb = data_pb2.ColumnFamily(name=column_family_name)
+        else:
+            request_pb = data_pb2.ColumnFamily(
+                name=column_family_name,
+                gc_rule=gc_rule.to_pb(),
+            )
+
+        # Create response_pb
+        response_pb = data_pb2.ColumnFamily()
```

Comment:
Similar to #1322/#1323. Unless my comment causes you to change something there, no need to address it here.